### PR TITLE
Add com.system76.Popsicle

### PR DESCRIPTION
**Pull request by ids1024**

### CODE REVIEW

File: com.system76.Popsicle.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- org.freedesktop.Sdk.Extension.rust-stable
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+  env:
+    CARGO_HOME: /run/build/popsicle/cargo
+finish-args:
+- --share=ipc
+- --socket=fallback-x11
+- --socket=wayland
+- --device=dri
```
Comment (bilelmoussaoui) [inline com.system76.Popsicle.yaml:15]:
```suggestion
```
already included in device=all.

File: com.system76.Popsicle.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- --device=all
+- --filesystem=home
```
Comment (bilelmoussaoui) [inline com.system76.Popsicle.yaml:17]: I wonder why this permission is necessary, GtkNativeFileChooser can't be used here?

**Comment by bilelmoussaoui** [comment]
bot, build com.system76.Popsicle

**Comment by nedrichards** [comment]
Is there anyone else from upstream who will need access when added?

File: com.system76.Popsicle.yaml (hunk 2 above)
Comment (ids1024) [reply com.system76.Popsicle.yaml:16]: Right. I don't think migrating Popsicle to using `GtkNativeFileChooser` would pose any difficulties...

Until very recently recently Popsicle couldn't run in a sandbox, and I don't believe it's been tested on Windows or Mac. So there wasn't any difference, but it looks like switching to `GtkNativeFileChooser` would be a good idea now.

**Comment by ids1024** [comment]
> Is there anyone else from upstream who will need access when added?

You can add @jackpot51 and @mmstick.

File: com.system76.Popsicle.yaml (hunk 2 above)
Comment (bilelmoussaoui) [reply com.system76.Popsicle.yaml:16]: Not a blocker, but it would be nice to land that in the future. Merged

Comment (ids1024) [reply com.system76.Popsicle.yaml:16]: Yep. That, and potentially using UDisks2 to find block devices so `--device=all` isn't needed.

Comment (bilelmoussaoui) [reply com.system76.Popsicle.yaml:16]: Ah didn't catch this one

**Pull request closed without merging**